protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -2609,10 +2609,43 @@ message ListLiquidityProvisionsResponse {
   LiquidityProvisionsConnection liquidity_provisions = 1;
 }
 
+// Liquidity provider commitment. For API purposes this can report a pending liquidity provision,
+// the current live provision or both in the event an update has been accepted on the network, but has yet
+// to go live and there is still a pending provision that will become active in the next epoch.
+message LiquidityProvision {
+  // ID for the liquidity provision.
+  string id = 1;
+  // Party ID for the creator of the provision.
+  string party_id = 2;
+  // Timestamp in Unix nanoseconds for when the liquidity provision was created.
+  int64 created_at = 3;
+  // Timestamp in Unix nanoseconds for when the liquidity provision was updated.
+  int64 updated_at = 4;
+  // Market ID for the liquidity provision.
+  string market_id = 5;
+  // Specified as a unitless number that represents the amount of the market's settlement asset.
+  // This field is an unsigned integer scaled to the asset's decimal places.
+  string commitment_amount = 6;
+  // Provider's nominated liquidity fee factor, which is an input to the calculation of taker fees on the market, as per setting fees and rewarding liquidity providers.
+  string fee = 7;
+  // Set of liquidity sell orders to meet the liquidity provision obligation.
+  repeated vega.LiquidityOrderReference sells = 8;
+  // Set of liquidity buy orders to meet the liquidity provision obligation.
+  repeated vega.LiquidityOrderReference buys = 9;
+  // Version of this liquidity provision.
+  uint64 version = 10;
+  // Status of this liquidity provision.
+  vega.LiquidityProvision.Status status = 11;
+  // Reference shared between this liquidity provision and all its orders.
+  string reference = 12;
+  // Liquidity provision that is currently pending and will go live in the next epoch.
+  optional vega.LiquidityProvision pending = 13;
+}
```

Comment:
Why not do something like this instead:
```
message LiquidityProvision {
   optional vega.LiquidityProvision active = 1 ;
   optional vega.LiquidityProvision pending = 2;
}
```

Reply:
I agree. There is probably no need for all this extra information on the top level

Reply:
Because I wanted to make the minimal breaking change possible. This additional field with the pending will only affect the gRPC API, the REST API should just have an additional field and wouldn't break I think.

Reply:
I don't think it would be a huge amount of work, no, up to you really.

Reply:
After quick discussion, we're going to go with the deprecation route.